core/nginx/conf/nginx.conf

```diff
@@ -84,7 +84,7 @@ http {
       # Favicon stuff
       root /static;
       # Variables for proxifying
-      set $admin {{ ADMIN_ADDRESS }};
+      set $admin {{ ADMIN_ADDRESS }}:8080;
```

Comment:
I fear this would break existing setups where people added a port on their own.
We would need to parse "ADMIN_ADDRESS" in socrate and add port 8080 if none is given.
This change would affect all other places you added ":8080"